app/src/main/java/org/stepik/android/domain/course_info/interactor/CourseInfoInteractor.kt

```diff
@@ -57,7 +57,7 @@ constructor(
             instructors    = (instructors ?: course.instructors?.map { null })?.takeIf { it.isNotEmpty() },
             language       = course.language,
             certificate    = course.certificate
-                ?.takeIf { course.isCertificateAutoIssued }
+                ?.takeIf { course.isCertificateAutoIssued || course.isCertificateIssued }
```

Comment:
It will be better to use the same condition as used in web version 
https://github.com/bioinf/edy/blob/81998b92d8116a88228243f356d3104ddf876534/apps/frontend/stepic/app/models/course.js#L273